resource-definitions/terraform-driver/credentials/README.md

```diff
@@ -11,13 +13,29 @@ A powerful approach for working with different cloud accounts for the same resou
 
 In this set of examples, we provide two `config` Resource Definitions for AWS and GCP.
 
-## AWS
+### AWS
 
 - [Account config (`account-config-aws.yaml`)](./account-config-aws.yaml)
 - [Provider Credentials (`aws-provider-credentials.yaml`)](./aws-provider-credentials.yaml)
 - [Environment Credentials (`aws-environment-credentials.yaml`)](./aws-environment-credentials.yaml)
 
-## GCP
+### GCP
 
 - [Account config (`account-config-gcp.yaml`)](./account-config-gcp.yaml)
-- [File Credentials (`gcp-file-credentials.yaml`)](./gcp-file-credentials.yaml)
\ No newline at end of file
+- [File Credentials (`gcp-file-credentials.yaml`)](./gcp-file-credentials.yaml)
+
+## Dynamic credentials
+
+Using a Cloud Account type that supports dynamic credentials, those credentials can be easily injected into a Resource Definition using the Terraform Driver. Use a `driver_account` referencing the Cloud Account in the Resource Definition, and access its the credentials through the supplied values as shown in the examples.
+
+### AWS
+
+- [S3 bucket (`s3-dynamic-credentials.yaml`)](./s3-dynamic-credentials.yaml)
```

Comment:
I had doubts here, should we combine `aws-provider-credentials.yaml` and `s3-dynamic-credentials.yaml` into a single example as they are quite similar.

But I decided to keep two section separately: `General credentials configuration` and `Dynamic credentials`, two demonstrate and emphasise different things. Wdyt?

Reply:
I think it's fine to leave both examples available. As you say, they demonstrate different approaches of obtaining the Cloud Account credentials, either directly or through a config resource. Wouldn't change it.